### PR TITLE
feat: update SDK to include new feed/parent_urls route

### PR DESCRIPTION
**Pull request by valerierose**
### Feed
`fetchFeedByParentUrls` -> https://docs.neynar.com/reference/feed-parent-urls

### CODE REVIEW

**Comment by manan19** [comment]
@Shreyaschorge can you review?

File: src/neynar-api/neynar-api-client.ts [hunk 1]
```diff
@@ -2312,7 +2343,7 @@ export class NeynarAPIClient {
   public async fetchNotificationsByParentUrlForUser(
     fid: number,
     parentUrls: string[],
-    options?: { cursor?: string; isPriority?: boolean }
+    options?: { cursor?: string }
```
Comment (Shreyaschorge) [inline src/neynar-api/neynar-api-client.ts:2346]: Same here

File: src/neynar-api/neynar-api-client.ts [hunk 2]
```diff
@@ -2030,7 +2064,7 @@ export class NeynarAPIClient {
    */
   public async fetchAllNotifications(
     fid: number,
-    options?: { cursor?: string; isPriority?: boolean }
+    options?: { cursor?: string }
```
Comment (Shreyaschorge) [inline src/neynar-api/neynar-api-client.ts:2067]: And here

**Review by Shreyaschorge — CHANGES_REQUESTED** [review]
@valerierose Minor changes.
Rest LGTM!

File: src/neynar-api/neynar-api-client.ts [hunk 3]
```diff
@@ -2065,7 +2097,7 @@ export class NeynarAPIClient {
   public async fetchChannelNotificationsForUser(
     fid: number,
     channelIds: string[],
-    options?: { cursor?: string; isPriority?: boolean }
+    options?: { cursor?: string }
```
[the remark this answers is not in the document]
Comment (valerierose) [reply src/neynar-api/neynar-api-client.ts:2100]: got it

**Pull request merged**